### DISCUSSION TITLE
Read sensor value via MQTT

### DISCUSSION THREAD

**Discussion by CodeKek** [arendst/Tasmota#20150] [Q&A] [answered]
Sorry for the simple question, but I need your help.
My Tasmota device (Device1) with analog sensor sends every 300 seconds the value:
`SENSOR = {"Time":"2023-12-03T15:56:16","ANALOG":{"A0":103}}`
How can I force it to send within this intervall?
For a relay module (Device2) `cmnd/Device2/POWER` sends the state of POWER, but
`cmnd/Device1/ANALOG` or `cmnd/Device1/A0` does not work.

Is it possible 
a) request to send the sensor data or
b) change the intervall time per MQTT command?

Thanks for your help!
Peter

**Comment by barbudor** [comment, marked as answer]
Interval time is the telemetry period. You can change it with the command [`Teleperiod`](https://tasmota.github.io/docs/Commands/#teleperiod).

You can also poll with command [`Status10`](https://tasmota.github.io/docs/Commands/#status)

**Reply by CodeKek** [reply]
Thank you it works fine
`cmnd/Device1/Status 10` 

Teleperiod works too, but 10 sec is too long.

**Reply by barbudor** [reply]
Bad solution

If you need fast reaction, do it on the device itself using Rules or Scripting-Language

**Reply by AgIgor** [reply]
use teleperiod now

**Reply by sfromis** [reply]
Telemetry data is not meant for fast reactions to things happening in a short time frame (the purpose for that mechanism is to follow slowly changing sensors like temperature). Still, barbudor already mentioned the command to poll data `Status 10` - the space is important to include for it to work. 

But indeed, use rules (etc) to pick up data at shorter intervals, but that's not very fast. Depending on the use case, there is also the option of configuring `ADC Button`, which is faster but not very flexible.
